.github/workflows/release-build-publish.yml

```diff
@@ -119,36 +119,50 @@ jobs:
 
           echo "VERSION=$VERSION" >> $GITHUB_ENV
 
+          IMAGE_ID=ghcr.io/${{ github.repository_owner }}/${{ matrix.project }}
+
+          DOCKER_TAGS=$IMAGE_ID:$VERSION
+
+          PRERELEASE=${{ github.event.release.prerelease }}
+          echo PRERELEASE=$PRERELEASE
+
+          if [ "$PRERELEASE" == "false" ]; then
+              DOCKER_TAGS=$DOCKER_TAGS,$IMAGE_ID:latest
+          fi
+
+          echo "DOCKER_TAGS=$DOCKER_TAGS" >> $GITHUB_ENV
+
+      - name: Setup QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+
+      - name: Available platforms
+        run: echo ${{ steps.qemu.outputs.platforms }}
```

Comment:
only used to print info?

Reply:
yeah, just for easy help debugging if we ever need to build to additional platforms. I copied that from the provenance repo and saw it in some examples, so just decided to keep it